app/players/dwarves/dwarf.py

```diff
@@ -0,0 +1,17 @@
+from app.players import player
+from abc import ABC
+
+
+class Dwarf(player.Player, ABC):
+    def __init__(self, nickname: str, favourite_dish: str) -> None:
+        super().__init__(nickname)
+        self._favourite_dish = favourite_dish
+
+        # def get_rating(self) -> int:
+        #     return 0
+        #
+        # def player_info(self) -> str:
+        #     return f"player is {self.nickname}"
```

Comment:
The `get_rating` and `player_info` methods are currently commented out. According to the task requirements, these methods should be implemented. `get_rating` should return an integer representing the player's rating, and `player_info` should return a string with player information. Please uncomment and implement these methods to ensure they function as expected.